### PR TITLE
Problems with DG.Wkt.toLatLngs

### PR DESCRIPTION
**Pull request by Trufi**
Work with not all wkt geometries.

For example: POINT, MULTIPOINT, MULTIPOLYGON

We need more tests for this feature.

### CODE REVIEW

**Comment by Trufi** [comment]
Поправил метод и добавил тестов.

**Comment by ilfa** [comment]
:+1:

File: src/DGWkt/DGWkt.js [hunk 1]
```diff
@@ -196,17 +196,32 @@ DG.Wkt.geoJsonLayer = function (data, opts) {
     return DG.geoJson(DG.Wkt.toGeoJSON(data), opts);
 };
 
+DG.Wkt._coordsToLatLngs = function (coords) {
+    if (DG.Util.isArray(coords) && !DG.Util.isArray(coords[0])) {
+        return [DG.GeoJSON.coordsToLatLng(coords)];
+    }
+
+
```
Comment (RLRR) [inline src/DGWkt/DGWkt.js]: Поправь двойной перенос строки и :+1:

**Comment by andymost** [comment]
@ilfa @Trufi May be we should say in doc that work only with:
 * point
 * multipoint
 * linestring
 * multilinestring
 * polygon
 * maultypoligin

**Pull request merged**